momepy/shape.py

```diff
@@ -1173,55 +1168,47 @@ def true_angle(a, b, c):
                 return True
             return False
 
+        def _calc(geom):
+            distances = []  # set empty list of distances
+            centroid = geom.centroid  # define centroid
+            points = list(geom.exterior.coords)  # get points of a shape
+            stop = len(points) - 1  # define where to stop
```

Comment:
Since we're only using `len(points)` twice here, I'm on the fence for declaring that value. I'd say one argument for it is consistency with the `_calc()` implementation in `Squareness` (if we go with my suggestion). You can be the tie-breaker here, @martinfleis.

Reply:
I don't mind either way, it is cheap.